.github/actions/build-avr/build.sh

```diff
@@ -0,0 +1,19 @@
+set -e
+
+# WORKSPACE is the git repo
```

Comment:
Is the `WORKSPACE` environment variable required by Arduino? It seems as though you are recreating environment variables that are available via GitHub's CI infrastructure (e.g. `GITHUB_WORKSPACE`).

https://docs.github.com/en/actions/creating-actions/dockerfile-support-for-github-actions#workdir

Reply:
That's correct, maybe I should just reuse GITHUB_WORKSPACE. The motivation was to be able to run this locally too, but I do agree no need for another variable.